stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -188,10 +225,10 @@ function externRetrieveSubscriptionParameters(Listener subscriberListener) retur
 } external;
 
 
-# Object representing the configuration for the WebSub Subscriber Service Endpoint.
+# Represents the configuration for the WebSub Subscriber Service.
```

Comment:
These are not service configs, but listener configs. Please correct other places as well
```suggestion
# Represents the configuration for the WebSub Subscriber Service Listener.
```